.eslintrc.js

```diff
@@ -16,6 +16,8 @@ module.exports = {
       alias: {
         map: [['types', './types/index.js']],
       },
+      // Use <rootDir>/tsconfig.json for typescript resolution
+      typescript: {},
```

Comment:
we should add TS plugins for ESLint, similar to: https://github.com/callstack/eslint-config-callstack#typescript

Reply:
TS plugins are included in react-native eslint configuration. https://github.com/facebook/react-native/blob/master/packages/eslint-config-react-native-community/index.js#L48